src/cli/index.js

```diff
@@ -8,7 +8,7 @@ import { existsFileSync } from '../util/exists';
 import parseConfig from './parseConfig';
 
 const cliOptions = parseArgv(process.argv.slice(2), true);
-const configOptions = parseConfig();
+const configOptions = parseConfig(cliOptions.opts);
 const defaultOptions = parseArgv([]);
 
 const options = _.defaults({}, cliOptions, configOptions, defaultOptions);
```

Comment:
I added a default value in `parseArgv`.  That should make it show up in `opts`, shouldn't it?